### DISCUSSION TITLE
Add banner or different background color for different project

### DISCUSSION THREAD

**Discussion by anne-ribbon** [supabase/supabase#12080] [Feature Requests]
Hello, 
thank you for the work you do on Supabase, I really enjoy working with it. 

In our organization we have one project for staging, and one project for production. 
The interface for both is exactly the same and I think it is really easy to change something in the production database, thinking that we are changing something in the staging database. 

It would be great to have different background or top banner colors either for all projects or for the "sensitive" projects. 

Thank you.

**Comment by Yusuf-Munir** [comment]
+1

**Reply by Brandoncapecci** [reply]
I know it might not match their design system as requested but a staging db and prod db really should **not** look this similar:

<img width="465" height="39" alt="Screenshot 2025-09-05 at 2 23 49 PM" src="https://github.com/user-attachments/assets/dc921cdb-81a5-435a-a279-2052c72501b3" />
<img width="477" height="42" alt="Screenshot 2025-09-05 at 2 24 05 PM" src="https://github.com/user-attachments/assets/21140abe-60f2-4b40-a624-516b8c17f3c0" />

There's got to be something they can do very easily on the UX front to distinguish them a bit better (alternatively have a distinct area of showing modifications to a db done explicitly through the supabase UI so wrongful modifications can be more easily audited/rolled back, better clarify the mechanisms to protect prod, etc).
